assets/admin_themes/Sea_Green/layout/body.scss

```
@@ -5,4 +5,16 @@ body {
     display          : flex;
     min-height       : 100vh;
     flex-direction   : column;
+}
+
+// Grid column sort controls (links with role="button"): keyboard focus ring follows text color
+.grid-view-ls a.sort-link[role='button']:focus-visible {
+    outline        : 2px solid currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)